src/App/VPSScan/NinjaGraph.hs

```diff
@@ -74,9 +74,8 @@ getNinjaDeps baseDir opts@NinjaGraphOpts{..} =
     Nothing -> BL.toStrict <$> generateNinjaDeps baseDir opts
     Just ninjaPath -> readNinjaDepsFile ninjaPath
 
-scanNinjaDeps :: (Has Diagnostics sig m) => NinjaGraphOpts -> ByteString -> m [DepsTarget]
-scanNinjaDeps NinjaGraphOpts{..} ninjaDepsContents =
```

Comment:
turns out the `NinjaGraphOpts` argument here was unused -- and ghc 8.10 picked this up 🎉

Reply:
Nice!